code/game/objects/items/control_wand.dm

```diff
@@ -32,12 +32,16 @@
 	switch(mode)
 		if(WAND_OPEN)
 			mode = WAND_BOLT
+			desc = "Remotely controls airlocks. Current mode: toggle bolts"
 		if(WAND_BOLT)
 			mode = WAND_EMERGENCY
+			desc = "Remotely controls airlocks. Current mode: toggle emergancy access"
 		if(WAND_EMERGENCY)
 			mode = WAND_SPEED
+			desc = "Remotely controls airlocks. Current mode: change door speed"
 		if(WAND_SPEED)
 			mode = WAND_OPEN
+			desc = "Remotely controls airlocks. Current mode: open doors"
```

Comment:
```suggestion
			desc = "Remotely controls airlocks. Current mode: open door"
```
If above (line 41) is "change _**door**_ speed" then this should be consistent with it.